### PR TITLE
Facebook

### PR DESCRIPTION
**Pull request by AaronVasquez**

### CODE REVIEW

File: api/config/initializers/cookies_serializer.rb [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :json
+# Rails.application.config.action_dispatch.cookies_serializer = :json
```
Comment (AaronVasquez) [inline api/config/initializers/cookies_serializer.rb]: Do we want to keep this commented out?

File: api/config/initializers/devise_token_auth.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@
   # This route will be the prefix for all oauth2 redirect callbacks. For
   # example, using the default '/omniauth', the github oauth2 provider will
   # redirect successful authentications to '/omniauth/github/callback'
-  # config.omniauth_prefix = "/omniauth"
```
Comment (AaronVasquez) [inline api/config/initializers/devise_token_auth.rb]: This too?

File: mobile/src/redux/index.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ const persistConfig = {
 const persistedReducer = persistReducer(persistConfig, modules)
 
 const middleware = [
-  process.env.NODE_ENV === `development` && createLogger(),
+  // process.env.NODE_ENV === `development` && createLogger(),
```
Comment (AaronVasquez) [inline mobile/src/redux/index.js]: This too?

File: api/config/initializers/cookies_serializer.rb (hunk 1 above)
Comment (EdmundMai) [reply api/config/initializers/cookies_serializer.rb]: yes

File: api/config/initializers/devise_token_auth.rb (hunk 2 above)
Comment (EdmundMai) [reply api/config/initializers/devise_token_auth.rb]: not being used

File: mobile/src/redux/index.js (hunk 3 above)
Comment (EdmundMai) [reply mobile/src/redux/index.js]: made the expo log unnecessarily noisy, couldn't see any of my logs

**Pull request merged**